01-Login/server.py

```diff
@@ -78,23 +64,21 @@ def decorated(*args, **kwargs):
         if constants.PROFILE_KEY not in session:
             return redirect('/login')
         return f(*args, **kwargs)
+
     return decorated
 
 
 # Controllers API
-@APP.route('/')
+@app.route('/')
 def home():
     return render_template('home.html')
 
 
-@APP.route('/callback')
+@app.route('/callback')
 def callback_handling():
-    resp = auth0.authorized_response()
-    if resp is None:
-        raise AuthError({'code': request.args['error'],
-                         'description': request.args['error_description']}, 401)
+    resp = auth0.authorize_access_token()
 
-    url = 'https://' + AUTH0_DOMAIN + '/userinfo'
+    url = AUTH0_BASE_URL + '/userinfo'
     headers = {'authorization': 'Bearer ' + resp['access_token']}
     resp = requests.get(url, headers=headers)
     userinfo = resp.json()
```

Comment:
Actually, you don't need to perform this request at all. there is a `id_token` in `authorize_access_token` if it is odic. There is built-in support for `id_token` parsing, check the example of Google:  https://github.com/authlib/loginpass/blob/master/loginpass/google.py#L37